test/functional/crud_spec.test.js

```diff
@@ -476,3 +479,19 @@ describe('CRUD v2', function() {
 
   generateTopologyTests(testSuites, testContext);
 });
+
+describe('CRUD unified', function() {
```

Comment:
were the CRUD specs and the other specs that aren't explicitly versioned API pulled in as part of the versioned API implementation? just curious about the updates that don't seem to be directly related to versioned API itself

Reply:
This comes from the backport of NODE-3074, which adds tests in the new unified format for `estimatedDocumentCount`.